Lib/ufo2ft/postProcessor.py

```diff
@@ -113,6 +113,10 @@ def process(
 
         if compileTrueTypeHinting and "glyf" in self.otf:
             self._compile_truetype_hinting()
+            # Force compilation of the font to avoid glyph name problems
```

Comment:
Here I had to resort to a kind of hack ... forcing the font to be compiled once by saving it and opening it again, otherwise the glyph names go out of sync and glyphs will disappear from the font.